indico/modules/rb_new/operations/bookings.py

```diff
@@ -255,3 +267,24 @@ def create_booking_for_event(room_id, event):
         return resv
     except NoReportError:
         flash(_("Booking could not be created. Probably somebody else booked the room in the meantime."), 'error')
+
+
+def get_active_bookings(limit=20, **filters):
+    start_date = filters.pop('start_dt')
+    last_reservation_id = filters.get('last_reservation_id')
+
+    cond = [ReservationOccurrence.start_dt > start_date]
+    if last_reservation_id is not None:
+        cond += [db.and_(db.cast(ReservationOccurrence.start_dt, db.Date) >= start_date,
+                         ReservationOccurrence.reservation_id > last_reservation_id)]
+
+    query = _bookings_query(filters)
+    query = (query.filter(db.or_(*cond))
+                  .order_by(ReservationOccurrence.start_dt,
```

Comment:
too much indentation; see other multiline queries for examples. also, merge with the previous statement

Reply:
yeah, true.. forgot about this